Documentation/ceph-csi-drivers.md

```diff
@@ -22,8 +22,9 @@ cluster.
    server](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/)
 3. An up and running Rook instance (see [Rook - Ceph quickstart
    guide](https://github.com/rook/rook/blob/master/Documentation/ceph-quickstart.md))
+   (e.g. kubectl create -f common.yaml, operator.yaml, cluster.yaml)
```

Comment:
Point number 3 there isn't really true though is it? (I realize you didn't write it here) Getting a running rook is an effect of running steps below so a running rook isn't really a prereq.

Reply:
Well, actually...  my experience is that running the cluster deploy first and then running and update with the CSI components is much more reliable.  It's also what I interpreted from the old documentation even though I realize it may be possible to just use the CSI steps without deploying the cluster first.

What prompted me to update these docs was the trouble I had getting reliable, repeatable and successful deployments using the existing docs and such.

Anyway, I guess the point being if you want to remove that line that's fair enough, but I do think there's still something not quite ready to just use the csi-operator to deploy the whole thing.

Reply:
Cool, looks like things are kicking off just going straight to operator-with-csi.yaml  I'll let it finish up and update the doc appropriately, thanks for pointing this out!